glide.yaml

```diff
@@ -1,10 +1,11 @@
 package: github.com/heketi/heketi
 import:
 - package: github.com/auth0/go-jwt-middleware
+  version: 36081240882bbf356af6efb152969e4b0bcf4456
```

Comment:
This commit is from 2019 and as @phlogistonjohn mentioned, it will bring in dgrijalva/jwt-go as its dependency. I suggest that the version be bumped up to v1.0.0. v1.0.0 uses the fork "github.com/form3tech-oss/jwt-go".